test/sagemaker_tests/pytorch/training/integration/sagemaker/test_distributed_operations.py

```diff
@@ -170,6 +170,9 @@ def test_smmodelparallel_mnist_multigpu_singlenode(ecr_image, instance_type, sag
     """
     Tests pt gpt2 command via script mode
     """
+    framework, framework_version = get_framework_and_version_from_tag(ecr_image)
+    if framework == "pytorch" and Version(framework_version) == Version("1.9.1"):
+        pytest.skip("Skipping the test for PT1.9")
```

Comment:
Please do not use `Version(variable_version) == Version("x.y.z")` in code. It is highly restrictive, and will cause the condition to work incorrectly the moment a new patch version is used.

Please always use `Version(framework_version) in SpecifierSet("==1.9.*")` instead.

Reply:
I agree, the `SpecifierSet("==1.9.*")` solution is a good practice. Thanks, will make the change.